learning_resources/models.py

```diff
@@ -25,6 +25,11 @@
 )
 from main.models import TimestampedModel, TimestampedModelQuerySet
 
+if TYPE_CHECKING:
+    from django.contrib.auth import get_user_model
+
```

Comment:
what does this code do? It doesnt seem like User is referenced anywhere in this file. Might be worth leaving a comment

Reply:
It's referenced in the types of some of the function arguments: https://github.com/mitodl/mit-learn/pull/2012/files/6a35cb4f70d5a4364f2eb25905f46b76d27b788a#diff-3e244d0725c2dbf2dfd444546f36377feb3dcb0bfee9838f50a971237def1e72R336